library/patient_tracker.inc.php

```diff
@@ -168,14 +168,14 @@ function is_checkout($option)
 #   2. If the tracker item does exist, but the encounter has not been set
 function is_tracker_encounter_exist($apptdate, $appttime, $pid, $eid)
 {
-  #Check to see if there is an encounter in the patient_tracker table.
-    $enc_yn = sqlQuery("SELECT encounter from patient_tracker WHERE `apptdate` = ? AND `appttime` = ? " .
-                      "AND `eid` = ? AND `pid` = ?", array($apptdate,$appttime,$eid,$pid));
-    if ($enc_yn['encounter'] == '0' || $enc_yn == '0') {
-        return(false);
+    #Check to see if there is an encounter in the patient_tracker table.
+    $enc_yn = sqlQuery("SELECT encounter from patient_tracker WHERE `apptdate` = ? AND encounter > 0 " .
+        "AND `eid` = ? AND `pid` = ?", array($apptdate, $eid, $pid));
+    if (!$enc_yn['encounter'] || $enc_yn === false) {
+        return (0);
     }
 
-    return(true);
+    return ($enc_yn['encounter']);
```

Comment:
ahhh, now it makes more sense :)

Reply:
will this change cause issues with the equivalent query in the below manage_tracker_status() function ?